chart/values.yaml

```
@@ -567,6 +567,9 @@ pgbouncer:
   # Maximum clients that can connect to pgbouncer (higher = more file descriptors)
   maxClientConn: 100
 
+  # supply name of existing secret with pgbouncer.ini and users.txt defined
+  pgbouncerConfigSecretName: ~
```

```suggestion
  configSecretName: ~
```

Since it is already in `pgbouncer` key -- adding `pgbouncer` is a bit redundant. Can you also give an example of an existing config with `pgbouncer.ini` and `users.txt`